README.md

```
@@ -13,9 +16,99 @@
+
+## 📋 Table of Contents
+
+- [Why create-next-quick?](#-why-create-next-quick)
+- [Features](#-features)
+- [Quick Start](#-quick-start)
+- [Usage](#-usage)
+  - [Basic Usage](#basic-usage)
+  - [Interactive Mode](#interactive-mode-for-existing-projects)
+  - [Interactive Prompts](#interactive-prompts)
+  - [Example Walkthrough](#example-walkthrough)
+- [Configuration Options](#configuration-options)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)